models/staging/tpdm/base/base_tpdm__evaluation_element_rating.sql

```diff
@@ -0,0 +1,27 @@
+with educator_prep_programs as (
+    {{ source_edfi3('evaluation_element_rating') }}
+),
+renamed as (
+    select
+        tenant_code,
+        api_year,
+        pull_timestamp,
+        last_modified_timestamp,
+        file_row_number,
+        filename,
+        is_deleted,
+
+        v:areaOfRefinement::string    as area_of_refinement,
+        v:areaOfReinforcement::string as area_of_reinforcement,
+        v:comments::string            as comments,
+        v:feedback::string            as feedback,
+        -- descriptors
+        {{ extract_descriptor('v:evaluationElementRatingLevelDescriptor::string') }} as evaluation_element_rating_level,
+        -- unflattened lists
+        v:elementRatingResults  as element_rating_results,
+        -- references
+        v:evaluationElementReference         as evaluation_element_reference,
+        v:evaluationObjectiveRatingReference as evaluation_objective_rating_reference
```

Comment:
Same question here, should we pull out identificators from references for association in the model?

Reply:
we have org_id in both reference, which one should we pull? should these org_ids be the same?

Reply:
I hope they are the same, but I'd like to suggest pull from `evaluationObjectiveRatingReference ` as the UML says EvaluationElementRating has assossiated with EvaluationObjectiveRating.
![image](https://github.com/user-attachments/assets/3c55b102-d59d-4dd0-b796-b9929519fb31)

Reply:
We discussed on daily stand up to leave it as it is, wi will be able to add `ed_org_id` on the next layer

Reply:
Hmmm...but other models created by not us have it on the base layer, e.g. \models\staging\edfi_3\base\base_ef3__course_offerings.sql

Reply:
Models created in this PR already have it on the base. Why we don't have to do the same for this one?

Reply:
`EvaluationElementRating` has two associations we did not know from which one should we take org_id
lets discuss on stand up again